tests/utils/config/config.go

```diff
@@ -82,6 +84,50 @@ func NewConfig() (*Config, error) {
 	return &conf, nil
 }
 
+func (c *Config) DebugTnf() (bool, error) {
+	if c.General.DebugTnf == "true" {
+		os.Setenv("LOG_LEVEL", "trace")
+
+		return true, nil
+	}
+
+	return false, nil
+}
+
+// CreateFolder verifies if a given folder exists, and creates it if it doesn't.
+func CreateFolder(folderPath string, perm fs.FileMode) (bool, error) {
+	_, err := os.Stat(folderPath)
+	if !os.IsNotExist(err) {
+		return false, err
+	}
+
+	err = os.MkdirAll(folderPath, perm)
+	if err != nil {
+		return false, err
+	}
+
+	return true, nil
+}
+
+func (c *Config) DefineLogFile(testSuite string, tcName string) *os.File {
+	folderPath := filepath.Join(c.General.ReportDirAbsPath, "Debug", testSuite)
+
+	_, err := CreateFolder(folderPath, 0755)
+
+	if err != nil {
+		panic(err)
+	}
+
+	tcFile := filepath.Join(folderPath, tcName+".log")
+	outfile, err := os.OpenFile(tcFile, os.O_APPEND|os.O_WRONLY|os.O_CREATE, 0755)
```

Comment:
You can just overwrite the content of the file  instead of deleting Debug folder in bash script.

Reply:
it will delete the content of the file in each launchTests func call, which means in each test run, that's not what we want.

Reply:
Done.